Tests/SwiftSyntaxMacrosTest/MacroSystemTests.swift

```diff
@@ -1037,4 +1058,39 @@ final class MacroSystemTests: XCTestCase {
       indentationWidth: indentationWidth
     )
   }
+
+  func testDeclsFromStringLiterals() {
+    assertMacroExpansion(
+      #"""
+      struct S {
+        public #decls(
+          """
+          static func foo() {
+          print("value") }
```

Comment:
Is this intentionally weirdly formatted?

Reply:
Yes, I wanted to confirm things are correctly expanded from string literals.

Reply:
OK 👍🏽